.github/workflows/anchore-scan.yml

```diff
@@ -0,0 +1,46 @@
+# This file is a modified version of a workflow file provided by GitHub, namely
+# the Security code-scanning workflow 'Anchore Grype Vulnerability Scan'.
```

Comment:
It says, 'This file is a modified version of a workflow file provided by GitHub.' I just wanted to clarify whether there is a need to update this workflow periodically to incorporate upstream changes?

Reply:
Good point. Yes one should periodically update GitHub Actions workflow files which depend on other Actions. For instance, this file uses the external `actions/checkout@v4` Action. When `actions/checkout@v5`is released this file should be updated. I suppose this is especially important for Actions which perform vulnerability scans. 

Regarding keeping the workflows up to date in practice, I don't know the best approach. I do know that you get a warning when running external Actions which have been deprecated. This will help inform maintainers when things go out of date.

Reply:
Is it worth mentioning this in the README file?

Reply:
Indeed it is. I will do this. Also, I need to check that the scan Action is actually the latest version, and update it if it is not.